sky/skylet/providers/gcp/config.py

```diff
@@ -824,6 +957,7 @@ def _configure_subnet(config, compute):
                     "type": "ONE_TO_ONE_NAT",
                 }
             ],
+            "tags": {"items": [config["cluster_name"]]},
```

Comment:
Why adding tags here?

Reply:
Because the firewall rules will only apply to the VM with the `cluster_name` tag. This add the `cluster_name` tag for each VM in the cluster

Reply:
Oh nvm... I thought you are saying L975-L980. This is redundant and it is removed now.